src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import './GoodsList.css';
+
+type Props = {
+  goods: string[],
+};
+
+export const GoodsList:React.FC<Props> = ({ goods }) => {
+  const [gooddies, setGoodies] = useState([...goods]);
```

Comment:
I think, that this line not obvious, and better write destruction for each function separately(setReverse, setSort,  seReset).